controllers/compliance.go

```diff
@@ -0,0 +1,108 @@
+package controllers
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/RedHatInsights/entitlements-api-go/config"
+	l "github.com/RedHatInsights/entitlements-api-go/logger"
+	"github.com/RedHatInsights/entitlements-api-go/types"
+	"github.com/getsentry/sentry-go"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+var screeningPathV1 = "/v1/screening"
+var complianceServiceName = "Export Compliance Service"
+
+var complianceFailure = promauto.NewCounterVec(
+	prometheus.CounterOpts{
+		Name: "it_export_compliance_service_failure",
+		Help: "Total number of IT export compliance service failures",
+	},
+	[]string{"code"},
+)
+var complianceTimeHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
+	Name:    "it_export_compliance_service_time_taken",
+	Help:    "Export compliance service latency distributions.",
+	Buckets: prometheus.LinearBuckets(0.25, 0.25, 20),
+})
```

Comment:
I thought it was simple enough as is but I'm very unfamiliar with the inner workings of prometheus so I'll defer to your judgement. Want me to throw a ticket in the backlog for this?

Reply:
It's probably fine as is. I was mostly considering if it would make sense to be able to configure the bucket values for reuse or adjustment via config vs hardcoded.